packages/studio-base/src/components/AccountSettingsSidebar/AccountInfo.tsx

```diff
@@ -16,7 +16,7 @@ import { useConfirm } from "@foxglove/studio-base/hooks/useConfirm";
 
 const log = Logger.getLogger(__filename);
 
-export const AVATAR_ICON_SIZE = 42;
+const AVATAR_ICON_SIZE = 42;
```

Comment:
Out of curiosity: why it was changed?

Reply:
This export was never used. It is shown when using the `yarn lint:unused-exports`